app/Console/Commands/TestCommand.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+
+class TestCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'command:name';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Command description';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this method "__construct" to simply inherit it. ([php:S1185](https://sonarcloud.io/organizations/dbmeneses-github/rules#rule_key=php%3AS1185))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=monica-dm-test&pullRequest=1&issues=AWSsghyll6vpliCetuqM&open=AWSsghyll6vpliCetuqM)

[ISSUE_KEY]: AWSsghyll6vpliCetuqM